src/routes/airtimeRoutes.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const { sendAirtime } = require('../controllers/airtimeControllers');
+
+const router = express.Router();
+
+router.post('/', sendAirtime);
```

Comment:
the post route will be `/airtime` so the full route will be `localhost:8080/api/v1/transfer/airtime` then the other for data and cash can follow the same pattern